examples/custom-routes-proxying/server.js

```diff
@@ -0,0 +1,11 @@
+const http = require('http')
```

Comment:
Let's remove this file, it should show proxying to an external website, potentially you can host a "legacy frontend" as an example

Reply:
Updated to use an existing site as the upstream proxy instead 👍